Tools/clinic/clinic.py

```diff
@@ -2044,23 +2046,30 @@ class Clinic:
 
 """
 
-    def __init__(self, language, printer=None, *, verify=True, filename=None):
+    def __init__(
+            self,
+            language: Language,
```

Comment:
Presumably this has to be a `CLanguage` instance. It's impossible to instantiate instances of `PythonLanguage`, since `PythonLanguage` does not implement the abstract method `render` that it inherits from the base class `Language`.

Reply:
Yes, thanks! Sorry for the sloppy PRs lately!